pyne/ensdf_processing.py

```diff
@@ -104,3 +104,105 @@ def gabs(inputdict_unchecked):
     proc.communicate()[0]
     proc.stdin.close()
 
+def logft(inputdict_unchecked):
+    #NOTE: changed input file line length to 90 to support longer file paths in fortran source.
+    """
+    This program calculates log ft values for beta and electron-capture decay, average beta energies, 
+    and capture fractions.  (LOGFT readme)
+
+    Input Dictionary Required Key Pair Value:
+        input_data_set : path to input data file.
+        output_report : desired path to output report file.
+        data_table : path to data table.
+        output_data_set : desired path to output data set.
+
+    Output Dictionary Values:
+        Everything in input dictionary is returned if LOGFT completes successfully.
+    """
+    inputdict = {}
+    input_data_set = inputdict_unchecked['input_data_set']
+    output_report = inputdict_unchecked['output_report']
+    data_table = inputdict_unchecked['data_table']
+    output_data_set = inputdict_unchecked['output_data_set']
+
+    exe_path = path_to_exe('logft')
+    inp = input_data_set + '\n' + output_report + '\n' + data_table + '\n' + output_data_set + '\n'
+    proc = subprocess.Popen([exe_path],stdout=subprocess.PIPE,stdin=subprocess.PIPE)
```

Comment:
`subprocess.check_output()` with the `input` kwarg is a nicer options for this and the following lines.  Not a blocker, but unless you really need it you probably shouldn't be using Popen() directly.